src/v/cluster/partition.cc

```diff
@@ -1001,6 +1001,103 @@ partition::get_follower_metrics() const {
     return _raft->get_follower_metrics();
 }
 
+ss::future<> partition::unsafe_reset_remote_partition_manifest(iobuf buf) {
+    vlog(clusterlog.info, "[{}] Unsafe manifest reset requested", ntp());
+
+    if (!(config::shard_local_cfg().cloud_storage_enabled()
+          && _archival_meta_stm)) {
+        vlog(
+          clusterlog.info,
```

Comment:
probably makes sense for this and the subsequent one to be warnings rather than info: if someone is calling this API and nothing's happening they'll want the messages to stick out

Reply:
Sure. The error bubbles up to the HTTP response, but I've lifted these to warn